shared/javascripts/local_storage.js

```diff
@@ -26,7 +26,7 @@ var ls = {
     if ( ls.localStorageDefined ) {
       localStorage.setItem(key, value);
     } else {
-      ls.preferences.setCharPref(key, value);
+      ls.storage[key] = value;
```

Comment:
Sorry, I misunderstood your comment, you were asking if `simple-storage` had the setItem and getItem, but the documentation doesn't expose any such API.